src/platforms/javascript/common/sourcemaps/uploading/hosting-publicly.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: Hosting Publicly
 description: "Learn about hosting publicly with JavaScript, working behind a firewall, and securing access to source maps."
+sidebar_order: 3
 ---
 
 By default, Sentry will look for source map directives in your compiled JavaScript files, which are located on the last line and have the following format:
```

Comment:
An extra nudge in the right direction cant hurt. 😛

```suggestion
<Note>

The most reliable way to provide your source maps to Sentry is to [upload them](..), because it cuts down on network traffic and ensures the correct version of the code and source maps will be used.

</Note>

By default, Sentry will look for source map directives in your compiled JavaScript files. These directives are located on the last line and have the following format:
```